core/src/main/scala/cats/mtl/MonadChronicle.scala

```diff
@@ -0,0 +1,29 @@
+package cats
+package mtl
+
+import cats.data.Ior
+
+trait MonadChronicle[F[_], E] extends Serializable {
+  val monad: Monad[F]
+
+  def dictate(c: E): F[Unit]
+
+  def disclose[A](c: E)(implicit M: Monoid[A]): F[A]
```

Comment:
We can implement this in terms of dictate :)
```scala
def disclose[A](c: E)(implicit M: Monoid[A]): F[A] =
  dictate(c).as(M.empty)
```

Reply:
👍